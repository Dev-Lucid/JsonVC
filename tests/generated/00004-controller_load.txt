{"title":"","description":"","keywords":"jvc_controller_00004","test":"","author":"","js":"","prepend":[],"append":[],"replace":[]}