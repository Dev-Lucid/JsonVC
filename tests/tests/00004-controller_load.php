<?
$controller = jvc::controller('00004');
jvc::set_response('keywords',get_class($controller));
file_put_contents($output_path,json_encode($__jvc['response']));
?>