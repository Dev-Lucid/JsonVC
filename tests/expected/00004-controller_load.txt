{"title":"","description":"","keywords":"jvc_controller_00004","author":"","js":"","prepend":[],"append":[],"replace":[]}